c3runtime/instance.js

```diff
@@ -94,8 +93,7 @@
 			if (this._automaticSuspend && this._runtime._suspendCount > 0) this._runtime.SetSuspended(false);
 		}
 		SetCommercialBreakConstraint({ constrained }) {
-			this._commercialBreakPossible = !constrained;
-			this.Trigger(self.C3.Plugins.Avix_PokiSDK_ForC3.Cnds.OnCommercialBreakPossible);
+			//this.Trigger(self.C3.Plugins.Avix_PokiSDK_ForC3.Cnds.OnCommercialBreakPossible);
```

Comment:
Should this `console.warn` maybe so developers know they should remove calls to this function?

Reply:
No, because this code will never be reached